Styles/index.scss

```diff
@@ -0,0 +1,58 @@
+@import "variables";
+@import "begins";
+@import "roads";
+@import "destination";
+@import "credits";
+
+.header {
+  background-color: $--yellow;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 10px;
+}
+
+.item {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+body {
+  background-color: $--gray;
+}
+
+.border {
+  border-style: inset;
+  border-color: $--yellow;
+}
+
+.fondo1 {
+  background-image: url("../Assets/bghero.webp");
+}
+
+.nav-items {
+  display: flex;
+  gap: 5px;
+  padding: 0 20px;
+  li {
+    list-style: none;
+  }
+}
+
+.portfolio {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  li {
+    list-style: none;
+  }
+}
+
+.align {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
1) Same as the comment on the HTML, don't use numbers on the variables. Use descriptive names such as `main-screen`, or maybe `destinations-screen`. `main-button`, `secondary-button` or stuff like that

2) Use mixins, you can avoid repeating the same `display:flex` situation all over your code declaring a mixin once and replacing it thruought all the app.